app/frontend/src/components/RequestForm.vue

```diff
@@ -170,14 +172,20 @@ export default {
           });
       }
     },
-
+    resetForm() {
+      this.form.applicationAcronym = '';
+      this.form.comments = '';
+      this.form.from = this.tokenParsed.email;
+      this.form.idir = this.tokenParsed.preferred_username;
+    },
     resetState() {
       this.errorOccurred = false;
       this.registerSuccess = false;
     }
   },
   mounted() {
     this.resetState();
+    this.resetForm();
```

Comment:
looks good and all works. just wondered if doing resetForm() is the best way to load the data.form.idir and data.form.from. could be hard coded into the template(?) also you may not want to reset the form when a user navigates to another page and then back.

Reply:
User navigation reset is a minor issue as 2 of the 4 fields are read-only to the user, and the only "real" field we care about is the acronym. It's highly unlikely a user will navigate to a different page while working on this form, and even if they did, there really isn't that much for them to fill out anyways.

The reason the values are loaded on mount via a resetForm method is because this is the earliest in the component lifecycle that it can get the JWT information from the vuex auth layer.